velox/functions/sparksql/tests/ArithmeticTest.cpp

```diff
@@ -268,6 +268,17 @@ TEST_F(ArithmeticTest, cosh) {
   EXPECT_TRUE(std::isnan(cosh(kNan).value_or(0)));
 }
 
+TEST_F(ArithmeticTest, rint) {
+  const auto rint = [&](std::optional<double> a) {
+    return evaluateOnce<double>("rint(c0)", a);
+  };
+
+  EXPECT_EQ(rint(2.3), 2.0);
```

Comment:
I added the test case for `rint(2.5)`
> It would also be nice to add test cases for very large values where true nearest integer is simply not representable in 'double'.

`rint(std::nextafter(1.0, 0.0))` is added, I'm not sure I correctly understand this, please correct me if wrong.